tidb-configuration-file.md

```diff
@@ -372,10 +372,11 @@ The Plan Cache configuration of the `PREPARE` statement.
 
 - The number of cached statements.
 - Default value: `100`
+- The type is `uint`. Values less than 0 are converted to large integers.
```

Comment:
This is aligned in https://github.com/pingcap/docs/pull/3123/files#diff-c8655fccf1cdae606dc717101a6cf6b3R375